### PR TITLE
Mining Textures, Tweaks, & More!

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request
image
- Adds number of new mining-related textures to the game, alongside two hammer freebies! 
- jackhammers can now once again destroy walls and windows, albiet via. a long-lasting channel that fails if interfered with.
- Adds new "legacy" mining equipment based on old designs, flavored as outdated but reliable NT prototypes.
- Codelores the existence of the EXOCON corporation, a plucky band of ex-munition manufacturers who have taken up a domineering role the far frontier goods market post-conflict.
- Adds backpack loot to a few common legion spawns. 
- Adds a 100% original syndicate weapon design, please do not steal. (currently unobtainable, will be added to a ruin soon.)
- Significantly overhauls the syndicate Commando cyborg. Impending mapping will make this a unique loot item in a high-danger ruin, so the balance is a little funky right now.
-alters the dwayne and skipper slightly, moving their onboard mining equipment, adding an arms cabinet to the dwayne, and adding various small flavor changes to the dwayne.
- adds a hidden mining vendor board to the dwayne. It costs money to use now, so should be fairly balanced.
- fixes some assorted wording and spacing issues.
- fixes a runtime related to a clown simplemob.
- adds a few flavor showcases, to be used in an upcoming mapping project.
- adds some new corpses, usable in mapping.
-adds a mapping-spawnable nitroglycerin bottle. 
- removes a buggy and runtime-y crusher drop connected to ice demons, instead making them drop manually on kills(for now.)
- marg.

Ported content 'twas summoned forth from 
https://github.com/tgstation/tgstation/pull/62168
https://github.com/tgstation/tgstation/pull/59889
https://github.com/tgstation/tgstation/pull/50962

## Why It's Good For The Game

- Mmm new mining suit look like tasty rock mmm
- fixes some stuff I really should have noticed on passby 1
- dread, marg, for you have been made manifest
- jackhammers were objectively outmoded, and we needed a dedicated demolition tool, which is what jackhammers literally are irl

## Changelog
:cl:
add: The Syndicate Cleaver, a syndiepilled crusher option. You've never seen anything like it before. Ever. Don't make them sue.
add: legacy mining equipment, slightly more powerful versions of typical exploration gear that also cause mild slowdown.
add: new multi-functional equipment for the syndicate Brigador cyborg, as well as a new sprite.
add: varied references to EXOCON, a new-age frontier operations corp.
add: jackhammer wall and window smashing functionality. 
add: an exocon-themed secure crate.
add: some showcases, to be used in a mapping project.
tweak: set up a few new ss13 archetype corpses, usable in mapping
tweak: added some backpack loot to legion miner drops, and an Old Miner corpse drop.
balance: the legion staff now fires significantly weaker skulls.
fix: replaced ice demon crusher loot drops to prevent runtimes
fix: fixed a continuous clown simplemob-related runtime
imageadd: added a number of new mining-related sprites and recolors
imageadd: ported resprites of the singularity hammer and mjolnir
imageadd: added marg.
imageadd: an ominous wired-up brigador. Where could it possibly be used? find out on the next episode of dragon shiptest z
spellcheck: fixed some minor typos and spacing issues here and there
add: marg.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
link ported PRs

**Comment by Sun-Soaked** [comment]
if literally just adding a sprite to stationobj.dmi causes it to merge conflict again, my sanity will be irretrievably destroyed

**Comment by ZephyrTFA** [comment]
> if literally just adding a sprite to stationobj.dmi causes it to merge conflict again, my sanity will be irretrievably destroyed

You got a merge conflict because you based your branch on an older commit of master and when you made the PR to mege your branch in master wasn't on that commit any longer, as such its highly recommended that you merge master into your branch before making a PR <3

**Comment by ZephyrTFA** [comment]
TL;DR: the master branch is volatile and anytime another PR gets merged yours is almost certainly going to conflict

**Comment by Sun-Soaked** [comment]
alright, thanks for that
should I kill this one and start again from updated master, or is there a way to sweep this up cleanly?

> TL;DR: the master branch is volatile and anytime another PR gets merged yours is almost certainly going to conflict

**Comment by ZephyrTFA** [comment]
Since you are touching DMIs, the only easy way to ensure your sprites weren't overridden is to recreate your branch unfortunately.
git really only cooperates cleanly with text files

**Comment by Sun-Soaked** [comment]
righto, executing phoenix protocol
